packages/preprocess/src/process-queries.cjs

```diff
@@ -64,6 +64,23 @@ const createDefaultProps = function (filename, componentDevelopmentMode, duckdbQ
 				$: _${id}_query_text = \`${duckdbQueries[id].compiledQueryString.replaceAll('`', '\\`')}\`;
 				$: _${id}_has_unresolved = __checkForUnsetInputs\`${duckdbQueries[id].compiledQueryString.replaceAll('`', '\\`')}\`;
 
+				if (import.meta?.hot) {
+					import.meta.hot.on("evidence:queryChange", ({queryId, content}) => {
+						let errors = []
+						if (!queryId) errors.push("Malformed event: Missing queryId")
+						if (!content) errors.push("Malformed event: Missing content")
+						if (errors.length) {
+							console.warn("Failed to update query on serverside change!", errors.join("\\n"))
+							return
+						}
+
+						if (queryId === "${id}") {
+							_${id}_query_text = content
```

Comment:
Don't query files technically allow interpolated variables right now? I think this is technically a regression (though query files interpolating variables are kinda weird behaviour anyways)

Reply:
Ah yeah that's a good point; I think we can probably get away with "don't do that" for now, it is a pretty odd pattern